ToolsPage.py

```diff
@@ -629,7 +629,11 @@ def __init__(self, master):
 			("comment","str",    "", _("Comment")),
 			("feed",    "mm"  , 10., _("Feed")),
 			("feedz",   "mm"  ,  1., _("Plunge Feed")),
-			("stepz",   "mm"  ,  1., _("Depth Increment"))
+			("stepz",   "mm"  ,  1., _("Depth Increment")),
+			("zretract", "mm" ,   1.0, _("Z Distance before contact")),
+			("feedbeforecontact", "int" ,   80, _("Z Feed before contact %")),
+			("hardcrust", "mm" ,   1.0, _("Hard crust thickness")),
+			("hardcrustfeed",  "int", 70, _("Hard crust Z Feed %"))
```

Comment:
i really think that this will be only confusing to users without having real benefits